tera-templates/base.html

```diff
@@ -0,0 +1,43 @@
+{%- import "macros.html" as macros -%}
```

Comment:
These don't seem to be used in this template, only in templates that inherit from base. I'd rather import it explicitly there so it's clear that it's something we wrote and not part of tera.
```suggestion
```

Reply:
They're used throughout pretty much every template, plus macros require namespacing (`macros::whavever()`), so there's no confusion with tera